app/styles/module/_ads-porvata.scss

```diff
@@ -0,0 +1,177 @@
+
+$vpaid-z-index: 10;
+$controls-z-index: 11;
+
+.ad-slot {
+	.video-overlay {
+		@include absolute-fill();
+	}
+
+	.video-display-wrapper {
+		position: relative;
+
+		video, &.vpaid-enabled > div {
+			@include absolute-fill();
+		}
+
+		&.vpaid-enabled {
+			.video-player > div {
+				pointer-events: none;
+
+				video {
+					pointer-events: auto;
+				}
+			}
+
+			& > div {
+				left: 50%;
+				transform: translate(-50%, 0);
+			}
+		}
+
+		.video-player {
+			background: #000;
+			left: 50%;
+			margin: 0 auto;
+			pointer-events: auto;
+			transform: translate(-50%, 0);
+		}
+	}
+}
+
+.video-player {
+	position: relative;
+
+	video {
+		left: 0;
+	}
+
+	& > iframe {
+		z-index: -1;
+	}
+
+	.progress-bar {
+		background: $uap-progress-bar-color;
+		bottom: 0;
+		height: 8px;
+		left: 0;
+		position: absolute;
+		width: 100%;
+		z-index: $controls-z-index;
+	}
+
+	.current-time {
+		background: $uap-progress-bar-current-time-color;
+		height: 100%;
+		margin: 0;
+		max-width: 100%;
+		width: 0;
+		transition: width linear;
+		z-index: $controls-z-index;
+	}
+
+	.pause-overlay {
+		cursor: pointer;
+		height: 100%;
+		position: absolute;
+		top: 0;
+		width: 100%;
+		z-index: $controls-z-index;
+	}
+
+	.close-ad {
+		background-image: url($player-button-collapse);
+		cursor: pointer;
+		height: $player-button-size;
+		margin: $player-button-border-space $player-button-border-space $player-button-click-area $player-button-click-area;
+		position: absolute;
+		right: 0;
+		top: 0;
+		width: $player-button-size;
+		z-index: $controls-z-index;
+	}
+
+	.ima-mute-div, .volume-control {
+		bottom: 0;
+		padding: $player-button-click-area $player-button-border-space 10px $player-button-click-area;
+		position: absolute;
+		right: 0;
+		text-align: right;
+		z-index: $controls-z-index;
+
+		.speaker {
+			&.mute {
+				background-image: url($player-button-sound-off);
+			}
+
+			background-image: url($player-button-sound-on);
+			display: inline-block;
+			height: $player-button-size;
+			overflow: hidden;
+			position: relative;
+			width: $player-button-size;
+		}
+	}
+}
+
+.ad-slot.porvata-outstream {
+	&::before {
+		content: attr(data-label);
+		display: block;
+		font-size: 0.625rem;
```

Comment:
we can make it `.625rem;` as well as `.75rem;` below.

Why do we want to use `rem` here instead of px like in all other places?